medical_prescription_state_verify/models/medical_prescription_order.py

```diff
@@ -51,33 +38,30 @@ def write(self, vals, ):
             ValidationError: When a write is not allowed due to being in a
                 protected state
         """
-        for rec_id in self:
-            if rec_id.state_type == 'verified':
+        if self.stage_id.name == 'Verified':
```

Comment:
This will not work. We need an option on the stage to indicate it is a "verified" stage